test.css

```diff
@@ -6,26 +6,44 @@
 
   .specific-links {
     display: none;
+    /* width: 327px;
+    height: 56px; */
+    width: 327px;
+    height: 184px;
+    left: 24px;
+    top: 92px;
+    gap: 8px;
```

Comment:
- [ ] Also, as the previous reviewer suggested. Kindly follow the Figma design that is given. You have some places in your Popup menu that need to be fixed to match the design. Your design should match the given design. Please fix the design as well. Thank you!

Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/225342229-42c3a7e1-2bb1-4f0b-a2cc-16babfc7d1ab.png"> | <img src="https://user-images.githubusercontent.com/69011963/225342264-b8e59a41-f125-4357-91c4-43314a58a15e.png" width="1440"/>